export default {
    apps: {
        recentlyAdded: [
            {
                id: "d6bda456-46d4-11ea-a4de-008cfa5ae621",
                system_id: "de",
                name: "Copy of Rstudio-DESeq2",
                description: "Rstudio DESeq2 VICE image",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-02-03T22:30:53.789Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "a25da189-ffe6-4f61-9a91-704711f36bd7",
                system_id: "de",
                name: "New app",
                description: "asdf",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2014-12-08T06:01:37.242Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "9ed50f34-fa8f-11e9-9bfc-008cfa5ae621",
                system_id: "de",
                name: "Copy of sequenceserver",
                description: "sequenceserver test app",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-10-29T21:04:26.403Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "39896e30-3b5c-11e7-90ce-008cfa5ae621",
                system_id: "de",
                name: "Copy of New app",
                description: "asdf",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2017-05-17T23:54:54.624Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "b390f5f0-4f13-11e9-8949-008cfa5ae621",
                system_id: "de",
                name: "test-vice-delay",
                description:
                    "A test app for testing a tool that has a significant delay or amount of processing to perform in its entrpoint.",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-03-25T15:36:04.700Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "8bc52f58-4ff6-11e9-adb1-008cfa5ae621",
                system_id: "de",
                name: "wheee",
                description: "testing testing testing",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-03-26T18:39:53.693Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "d3ec0a42-6142-11e9-b580-008cfa5ae621",
                system_id: "de",
                name: "test-vice-potree-entwine",
                description: "testing",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-04-17T18:58:46.260Z",
                is_favorite: false,
                is_public: false,
            },
            {
                id: "a74b9840-7b23-11e9-86f2-008cfa5ae621",
                system_id: "de",
                name: "Copy of GEA RNA-Seq QC Demo 0.1",
                description:
                    "This is a demonstration application for the genomics education alliance RNA-Seq pipeline",
                wiki_url: null,
                integration_date: null,
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2019-05-20T17:21:07.223Z",
                is_favorite: false,
                is_public: false,
            },
        ],
        public: [
            {
                id: "0dfe3e35-0000-4995-bc31-24b94ee7eada",
                system_id: "de",
                name: "QA_App-Endpoint-Test-Mod5_QA",
                description: "Testing new app endpoints.  Modified.",
                wiki_url: null,
                integration_date: "2020-03-16T07:14:46.656Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-16T07:14:56.813Z",
                is_favorite: true,
                is_public: true,
            },
            {
                id: "c81731dc-6755-11ea-908b-c2a97b34bb42",
                system_id: "de",
                name: "QA_TestForMultiplePanelsTerrain_QA",
                description: "This is a new description for the shared app.",
                wiki_url: null,
                integration_date: "2020-03-16T07:14:35.636Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-16T07:14:35.636Z",
                is_favorite: true,
                is_public: true,
            },
            {
                id: "9e989f50-6109-11ea-ab9d-008cfa5ae621",
                system_id: "de",
                name: "Snakemake-VICE",
                description:
                    "VICE app with jupyter lab and snakemake-minimal. Also conda and RNAseq analysis tools",
                wiki_url: null,
                integration_date: "2020-03-10T21:48:05.803Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-10T21:48:05.803Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "b91ffda4-4df0-11ea-bd40-008cfa5ae621",
                system_id: "de",
                name: "HTSeqQC 1.0",
                description:
                    "HTSeqQC is an automated quality control analysis tool for a single and paired-end high-throughput sequencing data (HTS) generated from Illumina sequencing platforms.",
                wiki_url: null,
                integration_date: "2020-03-09T21:13:54.884Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-09T21:13:54.884Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "aee316c6-15d5-11ea-933d-008cfa5ae621",
                system_id: "de",
                name: "Sequence Properties Analyser",
                description:
                    "Compares statistical properties of two sets of protein sequences.",
                wiki_url: null,
                integration_date: "2020-03-04T18:40:22.199Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-04T18:40:22.199Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "58f783bc-5c95-11ea-b10d-008cfa5ae621",
                system_id: "de",
                name: "ALLMAPS",
                description: "Performs ALLMAPS merge and path",
                wiki_url: null,
                integration_date: "2020-03-02T15:36:48.063Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-02T15:36:48.063Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "6ff82cda-5c98-11ea-aad3-008cfa5ae621",
                system_id: "de",
                name: "ALLMAPS merge",
                description:
                    "ALLMAPS orders scaffolds by maximizing colinearity to a collection of genetic/genomic maps into the final chromosome build.  This is step one: merging maps into a bed file and generating the weights file. http://goo.gl/KI61Ow",
                wiki_url: null,
                integration_date: "2020-03-02T15:33:49.287Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-03-02T15:33:49.287Z",
                is_favorite: false,
                is_public: true,
            },
            {
                id: "a847402e-ff2a-11e9-815d-008cfa5ae621",
                system_id: "de",
                name: "QGIS Xpra Desktop",
                description:
                    "desktop running Xpra with QGIS, SAGA-GIS, GRASS, GDAL, PDAL",
                wiki_url: null,
                integration_date: "2020-02-17T22:20:28.291Z",
                username: "ipctest@iplantcollaborative.org",
                edited_date: "2020-02-17T22:20:28.291Z",
                is_favorite: false,
                is_public: true,
            },
        ],
    },
    analyses: {
        recent: [],
        running: [],
    },
    feeds: {
        news: [
            {
                id: "110 at https://cyverse.org",
                name:
                    "Researchers Go on Offense, Defense in Battle Against Hackers",
                description:
                    "Researchers Go on Offense, Defense in Battle Against Hackers\nlittin\nMon, 12/09/2019 - 14:57",
                link:
                    "https://cyverse.org/Researchers-Go-on-Offense-Defense-in-Battle-Against-Hackers",
                date_added: "2019-12-09T21:57:33.000Z",
            },
            {
                id: "88 at https://cyverse.org",
                name: "Researchers 'Light Up' the Plant Tree of Life",
                description:
                    "Researchers 'Light Up' the Plant Tree of Life\nlittin\nThu, 10/24/2019 - 08:42",
                link:
                    "https://cyverse.org/Researchers-Light-Up-the-Plant-Tree-of-Life",
                date_added: "2019-10-24T15:42:11.000Z",
            },
            {
                id: "65 at https://cyverse.org",
                name: "CyVerse ‘Power User’ Joins Boyce Thompson Institute",
                description:
                    "CyVerse ‘Power User’ Joins Boyce Thompson Institute\nlittin\nThu, 10/03/2019 - 00:27",
                link:
                    "https://cyverse.org/CyVerse-%E2%80%98Power-User%E2%80%99-Joins-Boyce-Thompson-Institute",
                date_added: "2019-10-03T07:27:35.000Z",
            },
            {
                id: "81 at https://cyverse.org",
                name:
                    "ECSS Program Accelerates XSEDE User's Career in Geoinformatics",
                description:
                    "ECSS Program Accelerates XSEDE User's Career in Geoinformatics\nmwall\nMon, 10/07/2019 - 15:36",
                link:
                    "https://cyverse.org/ECSS-Program-Accelerates-XSEDE-User%27s-Career-in-Geoinformatics",
                date_added: "2019-10-07T22:36:42.000Z",
            },
            {
                id: "5 at https://cyverse.org",
                name:
                    "Introducing DataHog: A Novel New App to Analyze Stored Files",
                description:
                    "Introducing DataHog: A Novel New App to Analyze Stored Files\nlittin\nMon, 08/05/2019 - 22:53",
                link:
                    "https://cyverse.org/Introducing%20DataHog%3A%20A%20Novel%20New%20App%20to%20Analyze%20Stored%20Files",
                date_added: "2019-08-06T05:53:13.000Z",
            },
            {
                id: "6 at https://cyverse.org",
                name: "Inaugural Open Science Course Teaches Best Practices",
                description:
                    "Inaugural Open Science Course Teaches Best Practices\nlittin\nMon, 08/05/2019 - 23:28",
                link:
                    "https://cyverse.org/Inaugural-Open-Science-Course-Teaches-Best-Practices",
                date_added: "2019-08-06T06:28:31.000Z",
            },
            {
                id: "54 at https://cyverse.org",
                name: "NSF 2026 Idea Machine: Reinventing Scientific Talent",
                description:
                    "NSF 2026 Idea Machine: Reinventing Scientific Talent\nlittin\nMon, 09/02/2019 - 21:17",
                link:
                    "https://cyverse.org/NSF-2026-Idea-Machine%3A-Reinventing-Scientific-Talent",
                date_added: "2019-09-03T04:17:03.000Z",
            },
            {
                id: "55 at https://cyverse.org",
                name: "UA Campus Comes Together to Celebrate Digital Literacy",
                description:
                    "UA Campus Comes Together to Celebrate Digital Literacy\nlittin\nMon, 09/02/2019 - 21:41",
                link:
                    "https://cyverse.org/UA-Campus-Comes-Together-to-Celebrate-Digital-Literacy",
                date_added: "2019-09-03T04:41:02.000Z",
            },
        ],
        events: [
            {
                id: "129 at https://cyverse.org",
                name: "Webinar: Workflow Management Using Snakemake",
                description:
                    "Webinar: Workflow Management Using Snakemake\n\n              \n      tinal\nMon, 03/02/2020 - 13:46\n\n            Webinar: Workflow Management Using Snakemake\n\n      \n            March 27, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (DST)\n\n      \n            About this Webinar:\n\nFrom where and how to get data for your analysis, to where and how to treat the outputs, workflow managers can help you achieve better scientific reproducibility and scalability. In this webinar, University of Arizona graduate student Sateesh Peri will demonstrate Snakemake, an open source workflow management tool. Sateesh will first identify use cases where workflow managers are helpful for automation so that you can understand Snakefile componenets, such as the rules, inputs, outputs and actions. Then he will lead us through how to write and run a Snakefile in CyVerse VICE (Visual and Interactive Compute Environment). Once you learn to properly use Snakemake (or similar workflow management tools), keeping track of and sharing your work becomes second nature, not only saving you time whenever you need to re-run all or part of an analysis but helping you reduce errors that naturally get introduced whenever a non-automated activity is done (i.e., as part of the human condition of doing computational science and not being a bot!).\n\nRegister Here\n\nAbout the Presenter: \nSateesh PeriSateesh Peri is a PhD candidate in Eric Lyon’s lab at the University of Arizona and also works at CyVerse as a graduate intern. His primary research focus is on identification and characterization of long non-coding RNAs (lincRNAs); in his spare time, Sateesh likes learning and teaching computational skills to biologists (which is why we love working with him at CyVerse!). He is an open-science evangelist and an active instructor for Data Carpentry and for CyVerse's Foundational Open Source Skills (FOSS) and Container Camp.",
                link:
                    "https://cyverse.org/webinar-workflow-management-using-snakemake",
                date_added: "2020-03-02T20:46:03.000Z",
            },
            {
                id: "124 at https://cyverse.org",
                name:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics",
                description:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics\n\n              \n      tinal\nTue, 01/28/2020 - 13:47\n\n            Webinar: Get Grounded in TERRA-REF: Publicly-accessible, Hi-res Sensor Data for Crop Phenomics\n\n      \n            March 13, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (Daylight Savings Time)\n\n      \n            Presentation Slides\n\n      \n            Webinar Video-recording:\n\n\n\nAbout this Webinar: \n\n\n\n\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\nThis webinar will introduce the TERRA-REF project that produced a large, high resolution public domain dataset to advance high throughput plant phenomics. The TERRA REF dataset contains remote sensing, plant trait, environmental, agronomic and genomic data. The dataset was produced to enable researchers to evaluate and develop new computational pipelines for converting sensor data into biological and agronomic understanding and solutions. David LeBauer, who led development of TERRA-REF's data and computing pipeline will describe the project along with its computing infrastructure and datasets. He will showcase the diverse suite of data generated by thermal, hyperspectral, color, laser 3D, and active photosynthetic fluorescence cameras as well as hand measurements, environmental sensors, and deep genomic sequencing.\n\nAnyone interested in plant sciences, robotics, statistics, remote sensing, data science, and how to make such data FAIR (findable, accessible, interoperable, and reusable) will benefit from learning about TERRA-REF's publicly-funded and publicly-accessible crop phenotype datasets. David will describe how the TERRA REF data was produced and how users can learn more and get started using the data and contributing new algorithms to high throughput phenomics pipelines.\n\nAbout the Presenter:\n\n\n\nDavid LeBauer is the director of Data Science for the Agricultural Experiment Station at the University of Arizona. His research is focused on using science to engineer more sustainable and productive crops and agricultural systems. To support this effort, he develops open software and data to integrate data and knowledge across disciplines. Key projects include the Predictive Ecosystem Analyzer (PEcAn) framework for data synthesis and forecasting with crop and ecosystem models and the TERRA Reference phenotyping database and computing pipeline.",
                link:
                    "https://cyverse.org/webinar-get-grounded-in-terra-ref-publicly-accessible-hi-res-sensor-data-for-crop-phenomics",
                date_added: "2020-01-28T20:47:52.000Z",
            },
            // fake a few more entries to stress the layout code a bit
            {
                id: "129 at https://cyverse.org",
                name: "Webinar: Workflow Management Using Snakemake",
                description:
                    "Webinar: Workflow Management Using Snakemake\n\n              \n      tinal\nMon, 03/02/2020 - 13:46\n\n            Webinar: Workflow Management Using Snakemake\n\n      \n            March 27, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (DST)\n\n      \n            About this Webinar:\n\nFrom where and how to get data for your analysis, to where and how to treat the outputs, workflow managers can help you achieve better scientific reproducibility and scalability. In this webinar, University of Arizona graduate student Sateesh Peri will demonstrate Snakemake, an open source workflow management tool. Sateesh will first identify use cases where workflow managers are helpful for automation so that you can understand Snakefile componenets, such as the rules, inputs, outputs and actions. Then he will lead us through how to write and run a Snakefile in CyVerse VICE (Visual and Interactive Compute Environment). Once you learn to properly use Snakemake (or similar workflow management tools), keeping track of and sharing your work becomes second nature, not only saving you time whenever you need to re-run all or part of an analysis but helping you reduce errors that naturally get introduced whenever a non-automated activity is done (i.e., as part of the human condition of doing computational science and not being a bot!).\n\nRegister Here\n\nAbout the Presenter: \nSateesh PeriSateesh Peri is a PhD candidate in Eric Lyon’s lab at the University of Arizona and also works at CyVerse as a graduate intern. His primary research focus is on identification and characterization of long non-coding RNAs (lincRNAs); in his spare time, Sateesh likes learning and teaching computational skills to biologists (which is why we love working with him at CyVerse!). He is an open-science evangelist and an active instructor for Data Carpentry and for CyVerse's Foundational Open Source Skills (FOSS) and Container Camp.",
                link:
                    "https://cyverse.org/webinar-workflow-management-using-snakemake",
                date_added: "2020-03-02T20:46:03.000Z",
            },
            {
                id: "130 at https://cyverse.org",
                name:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics",
                description:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics\n\n              \n      tinal\nTue, 01/28/2020 - 13:47\n\n            Webinar: Get Grounded in TERRA-REF: Publicly-accessible, Hi-res Sensor Data for Crop Phenomics\n\n      \n            March 13, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (Daylight Savings Time)\n\n      \n            Presentation Slides\n\n      \n            Webinar Video-recording:\n\n\n\nAbout this Webinar: \n\n\n\n\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\nThis webinar will introduce the TERRA-REF project that produced a large, high resolution public domain dataset to advance high throughput plant phenomics. The TERRA REF dataset contains remote sensing, plant trait, environmental, agronomic and genomic data. The dataset was produced to enable researchers to evaluate and develop new computational pipelines for converting sensor data into biological and agronomic understanding and solutions. David LeBauer, who led development of TERRA-REF's data and computing pipeline will describe the project along with its computing infrastructure and datasets. He will showcase the diverse suite of data generated by thermal, hyperspectral, color, laser 3D, and active photosynthetic fluorescence cameras as well as hand measurements, environmental sensors, and deep genomic sequencing.\n\nAnyone interested in plant sciences, robotics, statistics, remote sensing, data science, and how to make such data FAIR (findable, accessible, interoperable, and reusable) will benefit from learning about TERRA-REF's publicly-funded and publicly-accessible crop phenotype datasets. David will describe how the TERRA REF data was produced and how users can learn more and get started using the data and contributing new algorithms to high throughput phenomics pipelines.\n\nAbout the Presenter:\n\n\n\nDavid LeBauer is the director of Data Science for the Agricultural Experiment Station at the University of Arizona. His research is focused on using science to engineer more sustainable and productive crops and agricultural systems. To support this effort, he develops open software and data to integrate data and knowledge across disciplines. Key projects include the Predictive Ecosystem Analyzer (PEcAn) framework for data synthesis and forecasting with crop and ecosystem models and the TERRA Reference phenotyping database and computing pipeline.",
                link:
                    "https://cyverse.org/webinar-get-grounded-in-terra-ref-publicly-accessible-hi-res-sensor-data-for-crop-phenomics",
                date_added: "2020-01-28T20:47:52.000Z",
            },
            {
                id: "131 at https://cyverse.org",
                name: "Webinar: Workflow Management Using Snakemake",
                description:
                    "Webinar: Workflow Management Using Snakemake\n\n              \n      tinal\nMon, 03/02/2020 - 13:46\n\n            Webinar: Workflow Management Using Snakemake\n\n      \n            March 27, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (DST)\n\n      \n            About this Webinar:\n\nFrom where and how to get data for your analysis, to where and how to treat the outputs, workflow managers can help you achieve better scientific reproducibility and scalability. In this webinar, University of Arizona graduate student Sateesh Peri will demonstrate Snakemake, an open source workflow management tool. Sateesh will first identify use cases where workflow managers are helpful for automation so that you can understand Snakefile componenets, such as the rules, inputs, outputs and actions. Then he will lead us through how to write and run a Snakefile in CyVerse VICE (Visual and Interactive Compute Environment). Once you learn to properly use Snakemake (or similar workflow management tools), keeping track of and sharing your work becomes second nature, not only saving you time whenever you need to re-run all or part of an analysis but helping you reduce errors that naturally get introduced whenever a non-automated activity is done (i.e., as part of the human condition of doing computational science and not being a bot!).\n\nRegister Here\n\nAbout the Presenter: \nSateesh PeriSateesh Peri is a PhD candidate in Eric Lyon’s lab at the University of Arizona and also works at CyVerse as a graduate intern. His primary research focus is on identification and characterization of long non-coding RNAs (lincRNAs); in his spare time, Sateesh likes learning and teaching computational skills to biologists (which is why we love working with him at CyVerse!). He is an open-science evangelist and an active instructor for Data Carpentry and for CyVerse's Foundational Open Source Skills (FOSS) and Container Camp.",
                link:
                    "https://cyverse.org/webinar-workflow-management-using-snakemake",
                date_added: "2020-03-02T20:46:03.000Z",
            },
            {
                id: "132 at https://cyverse.org",
                name:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics",
                description:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible Hi-res Sensor Data for Crop Phenomics\n\n              \n      tinal\nTue, 01/28/2020 - 13:47\n\n            Webinar: Get Grounded in TERRA-REF: Publicly-accessible, Hi-res Sensor Data for Crop Phenomics\n\n      \n            March 13, 2020 | Virtual\n\n10am Pacific | 11am Mountain | 12noon Central | 1pm Eastern (Daylight Savings Time)\n\n      \n            Presentation Slides\n\n      \n            Webinar Video-recording:\n\n\n\nAbout this Webinar: \n\n\n\n\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\n/*-->*/\n\nThis webinar will introduce the TERRA-REF project that produced a large, high resolution public domain dataset to advance high throughput plant phenomics. The TERRA REF dataset contains remote sensing, plant trait, environmental, agronomic and genomic data. The dataset was produced to enable researchers to evaluate and develop new computational pipelines for converting sensor data into biological and agronomic understanding and solutions. David LeBauer, who led development of TERRA-REF's data and computing pipeline will describe the project along with its computing infrastructure and datasets. He will showcase the diverse suite of data generated by thermal, hyperspectral, color, laser 3D, and active photosynthetic fluorescence cameras as well as hand measurements, environmental sensors, and deep genomic sequencing.\n\nAnyone interested in plant sciences, robotics, statistics, remote sensing, data science, and how to make such data FAIR (findable, accessible, interoperable, and reusable) will benefit from learning about TERRA-REF's publicly-funded and publicly-accessible crop phenotype datasets. David will describe how the TERRA REF data was produced and how users can learn more and get started using the data and contributing new algorithms to high throughput phenomics pipelines.\n\nAbout the Presenter:\n\n\n\nDavid LeBauer is the director of Data Science for the Agricultural Experiment Station at the University of Arizona. His research is focused on using science to engineer more sustainable and productive crops and agricultural systems. To support this effort, he develops open software and data to integrate data and knowledge across disciplines. Key projects include the Predictive Ecosystem Analyzer (PEcAn) framework for data synthesis and forecasting with crop and ecosystem models and the TERRA Reference phenotyping database and computing pipeline.",
                link:
                    "https://cyverse.org/webinar-get-grounded-in-terra-ref-publicly-accessible-hi-res-sensor-data-for-crop-phenomics",
                date_added: "2020-01-28T20:47:52.000Z",
            },
        ],
        videos: [
            {
                id: "yt:video:2GNs2cbhI9Q",
                name:
                    "Webinar: Using VICE DESeq2 for RNA Differential Expression Analysis",
                description:
                    "As RNA sequencing becomes more affordable, differential expression analyses, which offer invaluable insights into intrinsic cellular processes and stochastic gene expression in biological and biomedical research, have become popular. In this webinar, University of Arizona doctoral candidate Sateesh Peri demonstrates how to use two handy new apps in the CyVerse Discovery Environment: the Read Mapping and Transcript Assembly (RMTA) app and the DESeq2 app (RStudio). For anyone doing RNA-Seq differential expression analyses, using these apps in CyVerse's Visual Interactive Computing Environment (VICE) enables you to easily launch RStudio, have quick access to your Data Store files, and interact with your analyses on the fly.\n\nDESeq-2 Video Timestamps - Sateesh Peri\n\n00:00  Introduction\n02:30 Webinar Learning Objectives\n03:15 RNA Seq overview\n06:15 RNA Seq Pipeline in CyVerse\n06:55 Step 1: Getting Data\n07:40 Step 2: Using the RMTA App\n09:35 Step 3: Differential Expression analysis using DESeq-2 app\n10:35 Pipeline demo using test data\n25:15 Saving outputs\n28:00 Questions & Answers \n31:41 End\n\nQuick start of RNA-Seq analysis in DE here:  https://rnaseq-webinar-2019.readthedocs.io/en/latest/\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                link: "https://www.youtube.com/watch?v=2GNs2cbhI9Q",
                date_added: "2019-11-08T19:22:10.000Z",
                publication_date: "2019-11-08T19:22:10.000Z",
                author: "CyVerse.org",
            },
            {
                id: "yt:video:YytjDtergF4",
                name:
                    "Webinar: Building Your Favorite Software and Environments in the VICE Workbench",
                description:
                    "Want to work with and share your favorite analyses in RStudio, Shiny apps, or Jupyter Notebooks in the cloud? CyVerse VICE (Visual Interactive Compute Environment) is what you've been looking for! In this focused, 30-minute webinar, CyVerse Scientific Analyst Tyson Swetnam will show you how to access common platforms like RStudio and Jupyter, how to use CyVerse resources to build your own favorite packages and tools, and how to bring your new containers to the VICE workbench. By using these CyVerse resources, you and your collaborators can easily expand the scale of computation and share reproducible scientific analyses and results with your community.\n\nVICEWorkbench Video Timestamps - Tyson Swetnam\n0:00 Introduction\n03:05 Four Key Concepts\n07:00  Data Science Workbench\n10:45  CyVerse VICE Workbench Tools: Docker Containers\n13:00  Applications: Executable vs. Interactive  vs. OSG Apps\n21:00  Integrate your own tools\n24:15  Live Demonstration of tool integration in CyVerse\n35:00  Questions & Answers\n46:07  End\n\nWebinar materials, including interactive slides, are available here: https://www.cyverse.org/webinar-build-your-favorite-software-and-environments-in-the-vice-workbench\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                link: "https://www.youtube.com/watch?v=YytjDtergF4",
                date_added: "2019-12-07T18:56:22.000Z",
                publication_date: "2019-12-07T18:56:22.000Z",
                author: "CyVerse.org",
            },
            {
                id: "yt:video:K070WD73ocQ",
                name: "Webinar: Don't Let Your Data Drag You Down",
                description:
                    "Datasets are becoming larger and more complicated, with many people often contributing to and using the same data. Simultaneously, funders, publishers, and research institutions are beginning to require open data. These pressures can leave researchers and project managers stressed out and struggling to manage their data.\n\nNo worries, CyVerse has got your back! Data management expert Ramona Walls will take you through key features of the CyVerse platform that support managing your data throughout the entire data life cycle – all in one place! From easy bulk uploads, to permissions management, to using metadata, to getting Digital Object Identifiers (DOI), this webinar provides the best practices and helpful hints that any scientist needs to keep the data deluge at bay.\n\nWebinar materials are available here: https://cyverse.org/webinar-dont-let-your-data-drag-you-down\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                link: "https://www.youtube.com/watch?v=K070WD73ocQ",
                date_added: "2020-01-31T19:50:10.000Z",
                publication_date: "2020-01-31T19:50:10.000Z",
                author: "CyVerse.org",
            },
            {
                id: "yt:video:6_Oj66jHzgc",
                name:
                    "Webinar: Fall in Love with High Performance Computing at CyVerse",
                description:
                    "In honor of (and apologies to!) Valentine's Day, this webinar will help you fall in love with CyVerse's High Performance Computing (HPC) resources provided by the CyVerse team at the Texas Advanced Computing Center (TACC). Through your CyVerse account, you can access HPC resources to do analyses on your large datasets that require more intensive compute power using any of hundreds of existing apps or one that you create yourself. John Fonner of TACC's Life Science Computing group will walk through the HPC resources available through CyVerse, how and when to use them, and how you can extend and automate your workflows using the TACC APIs (application programming interfaces). For anyone with large datasets and limited compute resources, this webinar will open doors accessing and using CyVerse HPC resources to \"get the job done!\"\n\nWebinar materials, including interactive slides, are available here: https://cyverse.org/webinar%E2%80%93fall-in-love-with-high-performance-computing-at-cyverse\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                link: "https://www.youtube.com/watch?v=6_Oj66jHzgc",
                date_added: "2020-02-14T23:48:06.000Z",
                publication_date: "2020-02-14T23:48:06.000Z",
                author: "CyVerse.org",
            },
            {
                id: "yt:video:BCxZvwv4bBE",
                name:
                    "Webinar: Metabarcoding and Microbiomes in the ClassroomQIIME2 with DNA Subway",
                description:
                    "Research into microbial diversity and microbiomes has transformed the way we understand the environment, agriculture, and human health. Now, DNA Subway makes it easier to explore this unseen diversity in the classroom using the latest research tools in a simple interface.\n\nThe newest release of DNA Subway Purple Line is a streamlined, classroom-friendly version of the popular QIIME2 microbiome bioinformatics software. Purple Line supports large datasets (your own or provided examples) and guides educators and students through real analyses while simplifying complex tasks such as metadata validation and generation of interactive visualizations. Experienced QIIME users can export Purple Line outputs to use in custom analyses.\n\nThis one-hour webinar will covered Purple Line basics (topics below) and gets you started with Purple Line in the classroom, integration into course-based research (CUREs), and ways to generate your own data. We will also highlight future faculty DNA Barcoding and Metabarcoding for CURES Workshops.",
                link: "https://www.youtube.com/watch?v=BCxZvwv4bBE",
                date_added: "2020-02-26T21:47:59.000Z",
                publication_date: "2020-02-26T21:47:59.000Z",
                author: "CyVerse.org",
            },
            {
                id: "yt:video:piDrI0GK_yA",
                name:
                    "Webinar: VIBRANT: Automated Recovery and Annotation of Microbial Viruses from Genomic Sequences",
                description:
                    "Viruses that infect bacteria and archaea are central components in the microbial communities of both natural and human environments. If you have metagenomic assemblies or isolated genomic sequence data, and want to know how viruses may play a role in your research data, you should know about VIBRANT, a computational tool that uses sequencing data to identify, curate, annotate and estimate the quality of these viral sequences and can predict both free and integrated viruses. In this webinar, University of Wisconsin's Kristopher Kieft will give a baseline overview of the role viruses may play in biogeochemical cycling of nutrients during infections and will demonstrate how you can run VIBRANT on CyVerse, interpret the results and implement the analysis of viruses into your current genomics pipelines.\n\nWebinar materials, including interactive slides, are available here: https://cyverse.org/webinar-vibrant-automated-recovery-and-annotation-of-microbial-viruses-from-genomic-sequences\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                link: "https://www.youtube.com/watch?v=piDrI0GK_yA",
                date_added: "2020-02-28T21:16:15.000Z",
                publication_date: "2020-02-28T21:16:15.000Z",
                author: "CyVerse.org",
            },
            {
                id: "yt:video:epq-XaEYNmE",
                name:
                    "Webinar: Get Grounded in TERRA-REF: Publicly-accessible, Hi-res Sensor Data for Crop Phenomics",
                description:
                    "This webinar introduces the TERRA-REF project that produced a large, high resolution public domain dataset to advance high throughput plant phenomics. The TERRA-REF dataset contains remote sensing, plant trait, environmental, agronomic and genomic data. The dataset was produced to enable researchers to evaluate and develop new computational pipelines for converting sensor data into biological and agronomic understanding and solutions. David LeBauer, who led development of TERRA-REF's data and computing pipeline, describes the project along with its computing infrastructure and datasets. He showcases the diverse suite of data generated by thermal, hyperspectral, color, laser 3D, and active photosynthetic fluorescence cameras as well as hand measurements, environmental sensors, and deep genomic sequencing.\n\nAnyone interested in plant sciences, robotics, statistics, remote sensing, data science, and how to make such data FAIR (findable, accessible, interoperable, and reusable) will benefit from learning about TERRA-REF's publicly-funded and publicly-accessible crop phenotype datasets. David describes how the TERRA-REF data was produced and how users can learn more and get started using the data and contributing new algorithms to high-throughput phenomics pipelines.\n\nWebinar materials, including interactive slides, are available here: https://cyverse.org/webinar-get-grounded-in-terra-ref-publicly-accessible-hi-res-sensor-data-for-crop-phenomics\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                link: "https://www.youtube.com/watch?v=epq-XaEYNmE",
                date_added: "2020-03-13T18:36:44.000Z",
                publication_date: "2020-03-13T18:36:44.000Z",
                author: "CyVerse.org",
            },
            {
                id: "yt:video:8Zlx6FvtHUk",
                thumbnailUrl:
                    "https://i.ytimg.com/vi/8Zlx6FvtHUk/hqdefault.jpg",
                name: "Webinar: Workflow Management Using Snakemake",
                description:
                    "From where and how to get data for your analysis, to where and how to treat the outputs, workflow managers can help you achieve better scientific reproducibility and scalability. In this webinar, University of Arizona graduate student Sateesh Peri demonstrates Snakemake, an open source workflow management tool. Sateesh first identifies use cases where workflow managers are helpful for automation so that you can understand Snakefile components, such as the rules, inputs, outputs and actions. Then he leads us through how to write and run a Snakefile in CyVerse VICE (Visual and Interactive Compute Environment). Once you learn to properly use Snakemake (or similar workflow management tools), keeping track of and sharing your work becomes second nature, not only saving you time whenever you need to re-run all or part of an analysis but helping you reduce errors that naturally get introduced whenever a non-automated activity is done (i.e., you're a human, not a bot!).\n\nWebinar materials, including interactive slides, are available here: https://cyverse.org/webinar-workflow-management-using-snakemake\n\nCyVerse provides powerful infrastructure for computational research involving large datasets and complex analyses. Using CyVerse, teams can overcome scaling and complexity challenges to tackle questions that previously were unapproachable.\n\n**Get Started!**\nCreate your Account: http://cyver.se/account \nLearn about CyVerse: https://cyverse.org \nLearn how to use CyVerse: http://cyver.se/learn \n\n**Make CyVerse work for you!**\nUse hundreds of bioinformatics apps and manage data in a simple web interface\nLaunch the Discovery Environment: http://cyver.se/de \n\n**Create a custom cloud-based scientific analysis platform or use a ready-made one for your area of scientific interest.**\nLaunch Atmosphere: http://cyver.se/atmo \n\n**Securely store data for active analyses or sharing with your collaborators.**\nLaunch the Data Store: http://cyver.se/datastore \n\n**Access discoverable and reusable datasets with metadata features and functions.**\nLaunch the Data Commons: http://cyver.se/datacommons\n\n**Exchange, explore, and analyze biological images and their metadata.**\nLaunch BisQue: http://cyver.se/bisque \n\n**Follow us!**\nTwitter: http://cyver.se/twitter \nFacebook: http://cyver.se/Facebook \nLinkedIn: http://cyver.se/LinkedIn \nYouTube: http://cyver.se/YouTube \nGitHub: http://cyver.se/GitHub \n\nStill have questions?\nEmail info@cyverse.org\n\nThis material is based upon work supported by the National Science Foundation under Grant Nos. DBI-0735191, DBI-1265383, and DBI-1743442.",
                link: "https://www.youtube.com/watch?v=8Zlx6FvtHUk",
                date_added: "2020-03-30T06:34:44.000Z",
                publication_date: "2020-03-30T06:34:44.000Z",
                author: "CyVerse.org",
            },
        ],
    },
    instantLaunches: [
        {
            quick_launch_name: "instant-launch-test",
            quick_launch_creator: "wregglej@iplantcollaborative.org",
            added_by: "wregglej@iplantcollaborative.org",
            app_id: "d61d9a26-e921-11e9-8fe0-008cfa5ae621",
            app_disabled: false,
            quick_launch_description: "",
            is_public: true,
            added_on: "2021-03-15T14:07:29.609419Z",
            id: "73b99da4-85d2-11eb-831f-62d47aced14b",
            app_deleted: false,
            quick_launch_id: "0c01f86f-3a60-4477-b158-d4ede1990f9b",
            app_description:
                "Jupyter Lab based on jupyter/datascience-notebook with updated iJab plugin, jupyterlab_irods and SQL ",
            submission: {
                config: {
                    "d61e56f0-e921-11e9-8fe0-008cfa5ae621_d62788d8-e921-11e9-8fe0-008cfa5ae621": [],
                },
                name: "jupyter-lab-instant-launch",
                app_id: "d61d9a26-e921-11e9-8fe0-008cfa5ae621",
                system_id: "de",
                debug: false,
                output_dir: "/iplant/home/wregglej/analyses_qa",
                notify: true,
            },
            integrator: "upendra_35@iplantcollaborative.org",
            app_name: "JupyterLab-with-sql-1.0.9",
        },
    ],
};
